README.md

```diff
@@ -7,3 +7,22 @@ Contains function definitions for the [OpenVR](https://github.com/ValveSoftware/
 1. `git submodule update --init --recursive` (initial checkout only)
 1. `git submodule foreach git pull origin master` to update the submodule 
 1. `cargo build --features "buildtime_bindgen"` to update the bindings
+1. Apply the workaround for broken OpenVR ABIs, if required
```

Comment:
I think blacklisting these in bindgen and hand-writing the definitions in lib.rs would be less fragile; as it stands, this makes it very easy to miss the manual step in an update.

Reply:
Well, I guess the PR is an improvement and I'd let it slide for now :)

Reply:
Fair enough!